### DISCUSSION TITLE
Backrest showing backup successful but it isn't?

### DISCUSSION THREAD

**Discussion by workmatic** [garethgeorge/backrest#326] [General]
So please bare with me, admittedly very new to both restic and backrest (and cloud backups in general), so I will probably need to find the time to go back a few steps and get comfortable working directly with restic to backrest first - but with that out the way!

I have deployed backrest and configured everything the repo and plan. All looking good.

I test by hitting backup with one file sitting in the plan/file path. Backrest reports successful operation but on the web ui of the cloud provider (backblaze) no files are there.

There doesn't seem to be anywhere to see logs? I've added --verbose flag to try to see more about what's happening but seems an option to see logs not yet developed. I'll note I also tried testing with --dryrun flag to begin and maybe it's possible this has confused the state.

To test further though, I purposely changed the keyid and accesskey and yet still - backrest showing successful? I would have thought the connection to the cloud store should instantly error if incorrect.

Any help with what is going on here or how I can get better view of the logs would be tremendously helpful.

**Comment by garethgeorge** [comment]
I think the place to start is confirming that you've added your repo with the right URI -- it sounds like your backups might be going to some unexpected location on the local filesystem (or even inside the container if you're using docker)?

Other debug steps are you can click "View Logs" next to an operation to see the output / what might be happening.
